userspace/libsinsp/ifinfo.cpp

```diff
@@ -203,7 +203,7 @@ bool sinsp_network_interfaces::is_ipv4addr_in_local_machine(uint32_t addr, sinsp
 
 				for(auto it = clist->begin(); it != clist->end(); ++it)
 				{
-					if(it->second.m_container_ip == addr)
+					if(htonl(it->second.m_container_ip) == addr)
```

Comment:
Can we remove any mention to Kubernetes in the above comment? Since we should be supporting it now.

Reply:
fixed